src/style.css

```diff
@@ -0,0 +1,33 @@
+/* stylelint-disable no-missing-end-of-source-newline */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: flex-end;
+}
```

Comment:
```suggestion
.stars {
  display: flex;
}
```
lets make it work like this

Reply:
If i delete 10,11,12 - stars will display in reversed format. Didn't previous variant was correct? 
![image](https://user-images.githubusercontent.com/54063988/197867101-89fa2aae-5b7a-4647-b0fd-fe1118ae4f38.png)

Reply:
Oh, i found the decision :nth-child(-n+...)